lib/sorteios_web/controllers/session_controller.ex

```diff
@@ -50,6 +50,12 @@ defmodule SorteiosWeb.SessionController do
     end
   end
 
+  def delete(conn, _) do
+    conn
+    |> configure_session(drop: true)
```

Comment:
Hi, I'm curious why you used configure_session?

I was maybe thinking of [clear_session/2](https://hexdocs.pm/plug/Plug.Conn.html#clear_session/1) but I'm open to feedback

Reply:
I'll double check this again. I remember having some strange behaviour when trying this, but I might have just done something wrong. Will look into this a bit more, and provide feedback.

Reply:
I checked what was done for logout looking at the new phx.gen.auth additions. 
It looks like `configure_session(conn, renew: true)` and `clear_session/1` is used. See https://github.com/dashbitco/mix_phx_gen_auth_demo/pull/1/files#diff-28c37a2a5a2d64995f1482f63a688624695c4f12001389d11dbd03c1d86f5fbbR62

It works well, but let me know if you require any further changes to this.

Reply:
Wow, nice find!